eregs_core/static/regulations/js/source/models/meta-model.js

```diff
@@ -120,7 +120,12 @@ var MetaModel = Backbone.Model.extend({
         }
 
         if (typeof this.supplementalPath !== 'undefined') {
-            url += this.supplementalPath + '/';
+            if (this.supplementalPath == 'sidebar') {
+                url = '/sidebar_partial/'
+            }
+            else {
+                url += this.supplementalPath + '/';
```

Comment:
Are the other `supplementalPath` formats going to stay the same?

- `/partial/diff/`
- `/partial/search/`
- `/partial/definition/`